### PR TITLE
[WIP] [Parser] detect if files use JSX

### PR DESCRIPTION
**Pull request by DavideDeCenzo**
Associated Issue: #4694

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

Here's the Debugger's Testing doc
https://docs.google.com/document/d/1oBMRxV8A2ag2t22YsQOxTdEv0mXKzIg0tggJjRkU1S0/edit#.
Feel free to improve it!

### CODE REVIEW

File: src/workers/parser/getSymbols.js [hunk 1]
```diff
@@ -105,6 +106,7 @@ function extractSymbols(source: Source) {
   const identifiers = [];
   const classes = [];
   const imports = [];
+  const hasJSX = [];
```
Comment (jasonLaster) [inline src/workers/parser/getSymbols.js]: lets use a boolean here: `let hasJsx = false`

File: src/workers/parser/getSymbols.js [hunk 2]
```diff
@@ -122,6 +124,10 @@ function extractSymbols(source: Source) {
         });
       }
 
+      if (t.isJSXElement(path)) {
+        hasJSX.push(true);
```
Comment (jasonLaster) [inline src/workers/parser/getSymbols.js]: `hasJsx = true`

**Comment by DavideDeCenzo** [comment]
Ok now it should be better @jasonLaster

**Comment by jasonLaster** [comment]
Great Work!

**Pull request merged**